packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -1,207 +1,180 @@
-                        </StyledButton>
-                        <StyledButton
-                            // isDisabled={!device.connected}
-                            isDisabled={false}
-                            // TODO:
-                            // onClick={() =>
-                            //      signVerifyActions.sign(account.accountPath, signMessage)
-                            // }
-                        >
-                            <Translation id="TR_SIGN" />
-                        </StyledButton>
-                    </RowButtons>
-                </Sign>
-                <Verify>
-                    <Title>
-                        <Translation id="TR_VERIFY_MESSAGE" />
-                    </Title>
-                    <Row>
-                        <Input
-                            label={<Translation id="TR_ADDRESS" />}
-                            name="verifyAddress"
-                            value={signVerify.verifyAddress}
-                            onChange={handleInputChange}
-                            type="text"
-                            state={verifyAddressError ? 'error' : undefined}
-                            bottomText={verifyAddressError ? verifyAddressError.message : null}
-                        />
-                    </Row>
-                    <Row>
-                        <Textarea
-                            label={<Translation id="TR_MESSAGE" />}
-                            name="verifyMessage"
-                            value={signVerify.verifyMessage}
-                            onChange={handleInputChange}
-                            rows={4}
-                            maxRows={4}
-                            maxLength={255}
-                        />
+                        {page === 'sign' ? (
+                            <SignAddressInput
+                                name="path"
+                                label={<Translation id="TR_ADDRESS" />}
+                                account={selectedAccount.account}
+                                revealedAddresses={revealedAddresses}
+                                error={formErrors.path}
+                                data-test="@sign-verify/path"
+                                {...pathField}
+                            />
+                        ) : (
+                            <Input
+                                name="address"
+                                label={<Translation id="TR_ADDRESS" />}
+                                type="text"
+                                state={errorState(formErrors.address)}
+                                bottomText={formErrors.address}
+                                data-test="@sign-verify/select-address"
+                                {...addressField}
+                            />
+                        )}
                     </Row>
                     <Row>
                         <Textarea
```

Comment:
What about adding the placeholder `Will be generated after Signing` as it is in the design?
![image](https://user-images.githubusercontent.com/3729633/135427013-9b8a93c4-d9a3-4c5b-b33b-46c6fa6e41b5.png)

Reply:
https://github.com/trezor/trezor-suite/pull/4297/commits/43a8c4291aa250e4aa8a537b7de7e38581948364